public/index.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html lang="en-us">
   <head>
-      <title>Application Template | <%= process.env.SITE_NAME %></title>
+      <title>My Learner Records | <%= process.env.SITE_NAME %></title>
```

Comment:
I am wondering if the `My Learner Records` text is something we would translate in other MFEs. Might be worth checking some of the other MFE repos to confirm?

Reply:
Ok, I did not see translations for the titles in other MFEs.